src/components/label.ts

```diff
@@ -41,10 +41,10 @@ export module Components {
      * "right"]`. Defaults to `"center"`.
      * @returns {Label} The calling Label.
      */
-    public xAlign(alignment: string): Label {
+    public xAlign(alignment: string) {
```

Comment:
Probably not part of this PR? There are more issues regarding signatures. In particular, comments should not have types in them anymore